lib/ts/recipe/session/sessionClass.ts

```diff
@@ -52,7 +55,17 @@ export default class Session implements SessionContainerInterface {
 
     getSessionData = async (): Promise<any> => {
         try {
-            return await SessionFunctions.getSessionData(this.recipeImplementation, this.sessionHandle);
+            let querier = Querier.getNewInstanceOrThrowError(SuperTokens.getInstanceOrThrowError().isInServerlessEnv);
```

Comment:
You should change `SessionFunctions.getSessionData` function to do this logic that you have made, and continue to use `SessionFunctions.getSessionData` inside this function here.

Reply:
Done